view/field/index.ts

```diff
@@ -132,33 +158,53 @@ function useSpinButton(input: HTMLInputElement): void {
       }
     }
 
-    draggableRect = target.getBoundingClientRect()
+    draggableState.rect = target.getBoundingClientRect()
```

Comment:
Using simple variables like `draggableRect` & `draggableControl` will be better for JS compression here as well